src/components/ui/TaskFooter.jsx

```diff
@@ -3,8 +3,18 @@ import { ReactComponent as EditIcon } from 'assets/edit.svg';
 import { ReactComponent as DoneIcon } from 'assets/ok.svg';
 import Button from 'components/ui/Button';
 import propTypes from 'prop-types';
+import { deleteTodo } from 'redux/actions/TodoAction';
+import { useDispatch } from 'react-redux';
+
+export default function TaskFooter({ isCompleted = false, taskId }) {
+  const dispatch = useDispatch();
+
+  const deleteHandler = (e) => {
+    e.preventDefault();
```

Comment:
Best practice is to use full abbreviation of variable
```suggestion
  const deleteHandler = (event) => {
    event.preventDefault();
```

Reply:
Renamed it. thanks